frameworks/helloworld/src/main/dist/examples/config_update.yml

```diff
@@ -0,0 +1,90 @@
+name: {{FRAMEWORK_NAME}}
```

Comment:
That's fine with me, this seems like pretty basic functionality (which I feel is the closest barometer of what we mean these days by "sanity")